frontend/src/modules/history/components/Translation.js

```diff
@@ -138,6 +158,38 @@ export default class Translation extends React.Component<Props> {
                         />
                     </div>
                     <menu className='toolbar'>
+                    { index === 0 ? null :
+                        this.state.isDiffVisible ?
+                            // Hide Diff
+                            <Localized
+                                id='history-translation-hide-diff'
+                                attrs={{ title: true }}
+                            >
+                                <a
+                                    className='toggle-diff'
+                                    href='#'
```

Comment:
This appends `#` to the URL when clicked. Is there a better workaround then using another element, e.g. `<button>`?

Reply:
/cc @adngdb

Reply:
Nope, using a `<button>` is the correct way to solve this. `<a>` should only be used when you're actually changing the URL.